src/lib/components/SiteFooter/index.jsx

```diff
@@ -82,6 +91,21 @@ class SiteFooter extends React.Component {
     return mobileLinks;
   }
 
+  renderBusinessMarketingListItem() {
+    const { siteName, enterpriseMarketingLink } = this.props;
+    return enterpriseMarketingLink && (
+      <li>
+        <a href={this.renderEnterpriseMarketingSiteUrl(enterpriseMarketingLink)}>
+          <FormattedMessage
+            id="footer.site-footer.link.business"
+            defaultMessage="{siteName} for Business"
+            values={{ siteName }}
```

Comment:
Can you add a ``description`` field (usually above ``values``) with a note to help explain to the translator what's going on here?  (Assuming that "Foo for Business" is even something that gets translated... or is it more like a brand name that's always in English?)

Reply:
I realize this is existing code that you just moved... alas, movement draws attention.  ;-)

![](https://66.media.tumblr.com/fb0f6f355aca934d8978c9c1e1be31ae/tumblr_pj2onzPoTa1r9ulgpo5_r1_500.gif)

Reply:
Would something like `description="This is a link that will point to a business marketing page for a specified website"` suffice? (Not sure how wordy is too wordy)

Reply:
Yup, that would be fine.  Thanks!

Reply:
Awhile ago, I asked about i18n for "edX for Business" on the header in the marketing site (edx-mktg). The answer I remember getting was that brands should technically not be translated.

So, we might want to just ignore the i18n translation for the brand here anyways?